src/scenes/Projects/Overview/ProjectOverview.graphql

```diff
@@ -24,7 +24,28 @@ fragment ProjectOverview on Project {
       ...DisplayLocation
     }
   }
-  marketingLocation {
+  marketingCountry {
+    canRead
+    canEdit
+    value {
+      ...DisplayLocation
+    }
+  }
+  marketingRegion {
+    canRead
+    canEdit
+    value {
+      ...DisplayLocation
+    }
+  }
+  marketingCountryOverride {
+    canRead
+    canEdit
+    value {
+      ...DisplayLocation
+    }
+  }
+  marketingRegionOverride {
     canRead
     canEdit
     value {
```

Comment:
Same with all of these.
Maybe a generic one that can be reused:
```
fragment WillsLocation on Location {
  canRead
  canEdit
  value {
    ...DisplayLocation
  }
}
```

And then use it like this:
```
...
marketingRegion {
  ...WillsLocation
}

marketingRegionOverride {
  ...WillsLocation
}

marketingCountry {
  ...WillsLocation
}

marketingCountryOverride {
  ...WillsLocation
}

```